### PR TITLE
Add dropbox url explanation

### PR DESCRIPTION
**Pull request by wiktorze**

### CODE REVIEW

File: replicator-resources.qmd [hunk 1]
```diff
@@ -309,6 +309,18 @@ You can try the web-based uploader which will take any URL directly:
 
 ![](img/nuvolos-large-upload.png)
 
+When copying the dropbox URL, remember to change 'dl=0' to 'dl=1' inside the URL. This forces the download. For example:
+```
+https://www.dropbox.com/scl/fo/gumryf1zs8lwk5zq5udo7/ABrnRWk5w859IIibD-_QM1Q/Szumilo-20230708-R1?*dl=1*&preview=Niko+Szumilo+-+MS20230708-replication.zip&rlkey=ei2aozcysqcchaoiwwanjcg4s&subfolder_nav_tracking=1
```
Comment (floswald) [inline replicator-resources.qmd]: let's convert this into
```
https://www.dropbox.com/scl/fo/gumryf1zs8lwk5zq5udo7/ABrnRWk5w859IIib/Author-YYYYMMDD-R1?*dl=1*
```

I garbled up the URL and deleted the rest of the parameters which we don't need.

File: replicator-resources.qmd [hunk 2]
```diff
@@ -309,6 +309,18 @@ You can try the web-based uploader which will take any URL directly:
...
+```
+
+To rename the file, open the terminal and write
+```
+mv 'old_file_name' 'new_file_name'
+rm 'old_file_name'
```
Comment (floswald) [inline replicator-resources.qmd]: you don't need the `rm`

File: replicator-resources.qmd [hunk 3]
```diff
@@ -309,6 +309,18 @@ You can try the web-based uploader which will take any URL directly:
...
+```
+
+If the file is very large, move it to the large file storage that you can create in the 'Project Configuration' tab.
```
Comment (floswald) [inline replicator-resources.qmd]: this part is actually pretty hard to read what I did here. could you maybe change this bit into a more readable example please?  not much to change but remove the `(base)` stuff and shorten the super long name a bit (maybe refer to tab completion again - people are not suppose to type that thing...) thank

**Review by floswald — CHANGES_REQUESTED** [review]
thanks so much!

**Comment by ejdataeditor** [comment]
thanks a lot!

**Pull request merged**